metric_learn/base_metric.py

```diff
@@ -242,6 +242,118 @@ def transform(self, X):
     """
 
 
+class BilinearMixin(BaseMetricLearner, metaclass=ABCMeta):
```

Comment:
this is a detail but I would place this class after the Mahalanobis one, as it is currently less used in the package